src/docs/PML/PMLUserGuide.adoc

```diff
@@ -432,18 +432,41 @@ you can also trigger the deployment of a specific VM using the Resource Manager
 
 In the following subsections, we will illustrate the life cycle of a MaaS_ML instance, from starting the generic service instance,
 deploying a specific model, pausing it, to deleting the instance. We will also talk about how the life cycle of a MaaS_ML instance
-can be managed via three different ways in PML:
+can be managed via four different ways in PML:
 
-. <<Via Studio Portal>>
-. <<Via Service Automation Portal>>
-. <<Via Swagger UI>>
+. <<MaaS_ML Via Workflow Execution Portal>>
+. <<MaaS_ML Via Studio Portal>>
+. <<MaaS_ML Via Service Automation Portal>>
+. <<MaaS_ML Via Swagger UI>>
 
 In the description below, multiple tables represent the main variables that characterize the MaaS_ML workflows.
 In addition to the variables mentioned below, there is a set of generic variables that are common between all workflows
 which can be found in the subsection <<AI Workflows Common Variables>>.
 The management of the life cycle of MaaS_ML will be detailed in the next sub-sections.
 
-=== Via Studio Portal
+=== MaaS_ML Via Workflow Execution Portal
+
+Open the link:https://try.activeeon.com/automation-dashboard/#/portal/workflow-execution[Workflow Execution Portal].
```

Comment:
space after:

Reply:
it is an Adoc command to be able to add a link associated to Workflow Execution Portal